tests/test_models.py

```diff
@@ -107,6 +109,9 @@ def test_question_answertext_filter(tmp_path):
       </questions>
     </question_versions>
   </question_category>
+      </question_bank_entry>
+    </question_bank_entries>
+
```

Comment:
Let's update this in-line fixture to be a bit more correct in terms of representing the XML structure in `questions.xml` + indented consistently with others:

```python
    question_bank_xml = """
<?xml version="1.0" encoding="UTF-8"?>
<question_categories>
  <question_category>
    <question_bank_entries>
      <question_bank_entry id="1">
      <idnumber>1234</idnumber>
      <question_version>
        <question_versions>
          <version>1</version>
          <questions>
            <question id="1">
              <qtype>numerical</qtype>
              <plugin_qtype_numerical_question>
                <answers>
                  <answer>
                    <answertext>112</answertext>
                  </answer>
                </answers>
              </plugin_qtype_numerical_question>
            </question>
          </questions>
        </question_versions>
      </question_version>
      </question_bank_entry>
    </question_bank_entries>
  </question_category>
</question_categories>
    """.strip()
```